en/3-transaction/types.md

```diff
@@ -0,0 +1,284 @@
+| `0x01` | `IssueTransaction`      | *Deprecated* | `500`           | Issue assets from a registered native asset             |
+| `0x02` | `ClaimTransaction`      | *Active*     | `0`             | Claim GAS from unclaimed transaction outputs            |
+| `0x20` | `EnrollmentTransaction` | *Deprecated* | `1000`          | Enroll new candidate for a consensus node               |
+| `0x24` | `VotingTransaction`     | *Removed*    | `0`             | Vote for consensus node candidates                      |
+| `0x40` | `RegisterTransaction`   | *Deprecated* | `10000`         | Register a new native asset on the blockchain           |
+| `0x80` | `ContractTransaction`   | *Active*     | `0`             | Common transaction to transfer assets                   |
+| `0x90` | `StateTransaction`      | *Active*     | `0`             | Voting and registering for a consensus node candidate   |
+| `0xb0` | `AgencyTransaction`     | *Removed*    | `0`             | Place orders on the blockchain                          |
+| `0xd0` | `PublishTransaction`    | *Deprecated* | `500`           | Publish a smart contract to the blockchain              |
+| `0xd1` | `InvocationTransaction` | *Active*     | `0`             | Invoke a smart contract on the blockchain               |
+
+### MinerTransaction
+The first transaction in a block to distribute network fees. It is generated by a consensus node and it will collect and distribute all the network fees in that block.
+
+#### Additional attributes for a MinerTransaction:
+
+| Field   | Type   | Description                                       |
+|---------|--------|---------------------------------------------------|
+| `nonce` | uint32 | Random number to make the transaction hash unique |
+
+Example:
+
+```json
+{
+   "txid":"0xdbf3ad80e483c56b996fcc1264384f87a86a889d3cc2c79888200d347923f6c6",
+   "size":70,
+   "type":"MinerTransaction",
+   "version":0,
+   "attributes":[
+
+   ],
+   "vout":[
+      {
+         "n":0,
+         "asset":"0x602c79718b16e442de58778e148d0b1084e3b2dffd5de6b7b16cee7969282de7",
+         "value":"0.001",
+         "address":"AWHX6wX5mEJ4Vwg7uBcqESeq3NggtNFhzD"
+      }
+   ],
+   "vin":[
+
+   ],
+   "sys_fee":"0",
+   "net_fee":"0",
+   "scripts":[
+
+   ],
+   "nonce":960164232
+}
+```
+
+### ClaimTransaction
+Claim GAS from unclaimed transaction outputs
+
+#### Additional attributes for a ClaimTransaction:
+
+| Field    | Type  | Description                                          |
+|----------|-------|------------------------------------------------------|
+| `claims` | array | References to spent outputs in previous transactions |
+
+#### Claims
+A `claim` is a reference to a spent [`output`](transactions.md#outputs) (only for the NEO asset) in a previous transaction. Each transaction can have up to 65536 claim references. Each claim reference in the array of claims has a set of required attributes.
```

Comment:
"Each transaction can have up to 65536 claim references"
Does this limit exist?